users/urls.py

```diff
@@ -1,7 +1,8 @@
 from django.urls import path
 
-from users.views import FindIdView
+from users.views import FindIdView, FindPwView
 
 urlpatterns = [
     path('/findid', FindIdView.as_view()),
+    path('/findpw',  FindPwView.as_view()),
```

Comment:
```suggestion
    path('/find-id', FindIdView.as_view()),
    path('/find-password',  FindPwView.as_view()),
```
이어지는 단어는 '-' 로 구분해주세요.

Reply:
수정 완료 했습니다~